index.js

```diff
@@ -1,102 +1,28 @@
-/* eslint max-classes-per-file: ["error", 2] */
-const booksList = document.getElementById('books-list');
+import { Book, Books } from './modules/books.js';
+// eslint-disable-next-line import/extensions
+import { DateTime } from './node_modules/luxon/src/luxon.js';
+import {
```

Comment:
- [ ] please remove this eslint disablement line and fix eslint errors yourself.
[hint] Kindle install luxon library and import `DateTime` from t like that:
```javascript
import { DateTime } from "luxon";
```

Reply:
@Omar-Muhamad this project does is not using webpack or any web bundler so I can not import the luxon like that.